src/static_canvas.class.js

```diff
@@ -1746,6 +1762,10 @@
       }
       this.forEachObject(function(object) {
         object.dispose && object.dispose();
+        // animation module is still optional
+        if (fabric.runningAnimations) {
+          fabric.runningAnimations.cancelByTarget(object);
```

Comment:
not sure this is good enough because what if there's a nested object being animated under a group?

Reply:
Give a few minutes I'm writing a function for canvas

Reply:
redundant because it's covered by `object.dispose`

Reply:
see #7629

Reply:
oh ok i did not think of object.dispose right.

Reply:
i want to puke, but if someone finds a good idea to animate an object clippath, when object is disposed that needs to be cancelled to. i ll add that.

Reply:
no maybe the clippath is an edge case to explain. But if the clippath is shared across objects, its animation should be cacnceled manually.

Reply:
> i want to puke, but if someone finds a good idea to animate an object clippath, when object is disposed that needs to be cancelled to. i ll add that.

Funny stuff.

But I saw you merged with this line. It should be removed.

Reply:
oh ok your commit was adding the code to make this line useless.
But wasn't removing this line.
I'll do that